features/step_definitions/api_credentials_steps.rb

```diff
@@ -1,33 +1,35 @@
+# frozen_string_literal: true
+
 # API v1:
-Given /^(buyer "[^"]*") has user key "([^"]*)"$/ do |buyer, key|
-  buyer.bought_cinstance.update_attribute(:user_key, key)
+Given "{buyer} has user key {string}" do |buyer, key|
+  buyer.bought_cinstance.update!(user_key: key)
 end
 
 Then /^I should see (the application id of buyer "[^"]*")$/ do |value|
   step %(I should see "#{value}")
 end
 
 
-Then /^I should see (the user key of buyer "[^"]*")$/ do |value|
+Then "I should see {the_user_key_of_buyer}" do |value|
```

Comment:
I think it is clearer in this case to have a step definition of `"I should see the user key of buyer {string}`. If I read step definition it will not be clear that I need to insert "the user key of buyer" before the actual key.

It is unlike other steps like `{provider}` where it can be taken as a convention to type `provider "something"` or `user "something"`, `application "something"`, etc. Here it is not obvious IMO.

Reply:
Ok, so convention is to replace `_` with ` `. Like with `email_template`. I think it should be fine. Although for a simple string value, I'd personally would have avoided it.